mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -9,22 +9,2404 @@
 			"name": "schemas",
 			"item": [
 				{
-					"name": "JSON API schema",
-					"event": [
+					"name": "provider",
+					"item": [
+						{
+							"name": "package",
+							"item": [
+								{
+									"name": "GET package collection schema",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"id": "93293945-b442-4fea-9469-f263180e0bd2",
+												"exec": [
+													"pm.test(\"GET schema_parameters OK\", function () {",
+													"    pm.response.to.be.ok;",
+													"});",
+													"",
+													"pm.test(\"GET schema_parameters has JSON body\", function () {",
+													"    pm.response.to.have.jsonBody();",
+													"});",
+													"",
+													"setEnvironmentVariable(\"packageCollection\", replaceResponseRefWithName(pm.response.text()));",
+													"",
+													"function checkVariableExist(name){ return pm.environment.has(\"schema_\"+ name);}",
+													"",
+													"function setEnvironmentVariable(name, data){ pm.environment.set(\"schema_\"+ name, data) }",
+													"",
+													"function extractName(url){ return url.substring(url.lastIndexOf(\"/\") + 1, url.lastIndexOf(\".\")); }",
+													"",
+													"function replaceResponseRefWithName(text){ return text.replace(/\"\\$ref\":\"([^\"]+\\/)(?=.*\\.json)/g, \"\\\"$ref\\\":\\\"schema_\"); }",
+													"",
+													"function fetchSchema(url){",
+													"  url = url.replace(/\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}/, \"localhost\");",
+													"  const echoGetRequest = {",
+													"    url: url,",
+													"    method: 'GET',",
+													"    header: {",
+													"      'X-Okapi-Module-Id' : pm.variables.get(\"kb-ebsco-java-module-id\"),",
+													"      'X-Okapi-Tenant' : pm.variables.get(\"xokapitenant\")",
```

Comment:
Curious on the header for kb-ebsco-java-module-id -- could this be more dynamic? I see it is set in folio-vagrant.postman_environment -- could we query okapi in running environment to get specific module id? Since this will be executed in CI/CD pipeline when there are new released versions

Reply:
added request to retrieve and set `kb-ebsco-java-module-id` with version deployed

Reply:
Great -- thanks @natali-zaitseva